src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -2706,7 +2705,7 @@ def __init__(self, config):
         super().__init__(config)
 
         self.bert = BigBirdModel(config)
-        self.sequence_summary = SequenceSummary(config)
```

Comment:
This was a bug in PyTorch's BigBird. There are no `MultipleChoice` BigBird checkpoints on the hub yet, so it shouldn't be a big problem. Will add `"bug correction / backwards breaking" in the PR description though